### DISCUSSION TITLE
Add Black background function

### DISCUSSION THREAD

**Discussion by hexman1999** [rustdesk/rustdesk#7278] [General]
I suggest adding a feature that enables the black background on the remote client in order to reduce data usage and add this as an option that can be enabled or disabled

Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/5990

**Reply by hexman1999** [reply]
> https://github.com/rustdesk/rustdesk/pull/5990

Not acting like that from android to windows 10

Is that option has to be activated from the settings?
